docker-compose.prod.yml

```diff
@@ -20,7 +20,7 @@ services:
 
   mysql:
     # in production, we may want to use a managed database service
-    image: percona:5.7
+    image: mysql:5.7 # Sylius is fully working on mysql 8.0 version
```

Comment:
Why didn't you change the version to 8.0?

Reply:
There is no reason. We support both versions, but which is preferable for us - I don't know.

I can create additional PR with the version change, as the goal of PR was to clarify the db engine. Comment was extra 😄